lib/cn/csched_sp3_work.c

```diff
@@ -356,7 +365,7 @@ bool
 sp3_work_splittable(struct cn_tree_node *tn, const struct sp3_thresholds *thresh)
 {
     return !tn->tn_ss_joining && (cn_ns_kvsets(&tn->tn_ns) > 0) &&
-        (cn_ns_clen(&tn->tn_ns) >= tn->tn_split_size ||
+        (cn_ns_wlen(&tn->tn_ns) >= tn->tn_split_size ||
```

Comment:
Ran into it by accident, running kmt where the base size didn't exceept the split size, so the node would never split because clen was always below the split size.  GC couldn't keep up, so node just kept getting longer and longer...
And yes, if/when things cool down the node joins again.